modules/bot/Module.php

```diff
@@ -150,20 +175,52 @@ private function initialize($update, $botId)
                 'first_name' => $updateChat->getFirstName(),
                 'last_name' => $updateChat->getLastName(),
             ]);
+
             if (!$telegramChat->save()) {
                 return false;
             }
 
+            // Add chat administrators to db
+            if ($newChat && $updateChat->getType() != Chat::TYPE_PRIVATE) {
+                $administrators = $this->botApi->getChatAdministrators($updateChat->getId());
+
+                foreach ($administrators as $administrator) {
+                    $chatMember = new ChatMember();
```

Comment:
Use [link()](https://www.yiiframework.com/doc/api/2.0/yii-db-baseactiverecord#link()-detail) method to fill junction table instead of explicit creating new instance of `ChatMember` class.